_data/guides.yml

```diff
@@ -189,5 +201,21 @@ toc:
 
 - title: Administering Federation
   section:
-  - /docs/admin/federation/kubfed/
-  - docs/admin/federation/index.md
+  - docs/admin/federation/kubefed.md
+  - title: Federated Kubernetes Objects
+    section:
+    - docs/user-guide/federation/index.md
+    - docs/user-guide/federation/configmap.md
+    - docs/user-guide/federation/daemonsets.md
+    - docs/user-guide/federation/deployment.md
+    - docs/user-guide/federation/events.md
+    - docs/user-guide/federation/federated-ingress.md
+    - docs/user-guide/federation/namespaces.md
+    - docs/user-guide/federation/replicasets.md
+    - docs/user-guide/federation/secrets.md
+    - docs/federation/api-reference/README.md
```

Comment:
Also, the title isn't missing; there's just both an explicit title and an H1 heading, which looks terrible.